test_cli.py

```diff
@@ -1045,14 +1042,33 @@ def test(ctx):
     wait(sugared_remote_uuid)
     check_contains(sugared_remote_name, _run_command([cl, 'search', sugared_remote_name]))
     check_equals(sugared_remote_uuid, _run_command([cl, 'search', sugared_remote_name, '-u']))
-    check_equals('ready', _run_command([cl, 'info', '-f', 'state', sugared_remote_uuid]))
     check_equals('hello', _run_command([cl, 'cat', sugared_remote_uuid + '/stdout']))
 
     # Explicitly fail when a remote instance name with : in it is supplied
     _run_command(
         [cl, 'run', 'cat %%%s//%s%%/stdout' % (source_worksheet_full, name)], expected_exit_code=1
     )
 
+    # Multi-alias tests
```

Comment:
Not clear what "multi-alias" means...explain: "Multiple keys pointing to the same bundle"

Reply:
Updated!